plugins/lookup/hashi_vault.py

```diff
@@ -132,6 +132,7 @@
         - approle
         - aws_iam_login
         - jwt
+        - agent
```

Comment:
Add a note in the `description` stating when this is supported from.

Reply:
Thanks @Akasurde , could you be more specific?  I don't see any particular `description` fields that seem to have specific notes about the versions when an auth method is added.  I added a comment that seemed to match the versions of the added methods.  Let me know if this isn't what you were wanting.